packages/monaco-graphql/src/languageFeatures.ts

```diff
@@ -238,13 +241,13 @@ export class CompletionAdapter
     try {
       const resource = model.uri;
       const worker = await this._worker(model.uri);
-      const completionItems = await worker.doComplete(
+      const completionItems = await worker?.doComplete(
         resource.toString(),
         position,
       );
       return {
         incomplete: true,
-        suggestions: completionItems.map(toCompletion),
+        suggestions: completionItems?.map(toCompletion) || [],
```

Comment:
hm... completion LS should _always_ return an array! empty or not

Reply:
Ah, I should take out the changes in this file — I was seeing errors in the console for trying to access `doValidation` on null from the worker, but this isn't really related  to the main changes anyway

Reply:
found the actual fix for this in https://github.com/graphql/graphiql/pull/2654/commits/3659928b5f30e6ca7c695ebe6722185569c98693!